packages/preferences-controller/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.3.1]
+### Changed
```

Comment:
Missing this PR: https://github.com/MetaMask/core/pull/1577, which adds a `setIsIpfsGatewayEnabled` method and `isIpfsGatewayEnabled` state

Reply:
Added here: [2be1a3d](https://github.com/MetaMask/core/pull/1634/commits/2be1a3dc902bf8d1b2f3b472fbd329ff42f34879)